source/store/actions/dynamicFormData.ts

```diff
@@ -0,0 +1,103 @@
+import { Form } from '../../types/FormTypes';
+import { Case } from '../../types/CaseType';
+import { User } from '../../types/UserTypes';
+import { convertAnswerArrayToObject } from '../../helpers/DataStructure';
+
+const addIds = (descriptor: string[], questionId: string, formId: string) =>
```

Comment:
What kind of Ids are we adding? Could be more specific for it's use case since the function takes a questionId i guess addQuestionIds would be more correct :)

Reply:
Well, it also adds the formId, so that would not be quite correct.